src/components/entity/ha-chart-base.html

```diff
@@ -1,17 +1,25 @@
 <link rel='import' href='../../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../../bower_components/paper-icon-button/paper-icon-button.html'>
 
 <dom-module id="ha-chart-base">
   <template>
     <style>
       .chartHeader {
         padding: 6px 0 0 0;
+        display: table;
```

Comment:
So `table` is usually kinda a hack. Can you use flexbox instead?  https://css-tricks.com/snippets/css/a-guide-to-flexbox/